Assignment2/InvalidInput.java

```diff
@@ -0,0 +1,8 @@
+
+
+/*This Exception class is when user enters invalid input like negative numbers for length, breadth, radius of Shapes.*/
+public class InvalidInput extends Exception {
```

Comment:
Typically you should use or suffix a custom exception class with "Exception". So the class should be named "InvalidInputException".